정봉석/vite-project/Counter/CounterPage.jsx

```diff
@@ -0,0 +1,19 @@
+import App from "../src/App";
+import { useState } from "react";
+
+export default function Counter() {
+  const [counter, setcounter] = useState(0);
+  const increase = () => {
+    setcounter(counter + 1);
+  };
+  const decrease = () => {
+    setcounter(counter - 1);
+  };
```

Comment:
`setter` 함수를 잘 작성해주셨군요!
onClick에 직접 넣을 수도 있지만, 재사용성과 안정성을 위해 변환 함수를 작성하여 사용하는 방법은 좋은 방법입니다!

추가적으로 counter의 값을 보장하기 위해 동적 변화를 주는건 어떠신가요?
```suggestion
  const increase = () => {
    setcounter(prev => prev + 1);
  };
  const decrease = () => {
    setcounter(prev => prev - 1);
  };
```